packages/cli/src/config/index.ts

```diff
@@ -19,6 +19,9 @@ if (inE2ETests) {
 		EXTERNAL_FRONTEND_HOOKS_URLS: '',
 		N8N_PERSONALIZATION_ENABLED: 'false',
 	};
+}
+if (inTest) {
+	process.env.N8N_PUBLIC_API_DISABLED = 'true';
```

Comment:
This helps with cutting down the heap usage in tests by about 15%, which in turn helps avoid OOM issues.